.copyright.hook

```diff
@@ -9,7 +9,7 @@ import subprocess
 import platform
 
 COPYRIGHT = '''
-  Copyright (c) 2016 PaddlePaddle Authors. All Rights Reserved.
+  Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserved.
```

Comment:
I think this is incorrect. In my opinion, it should be 2016. This is because, we only want to have the year as 2018 for those files which are created in the year 2018. 
If you look at the code, https://github.com/PaddlePaddle/Paddle/blob/develop/.copyright.hook#L39-L42, I think the intention here is to just change those files who are created in 2018. Moreover, we don't want to change the files which were created in 2016/2017. 

I think I will wait for @dzhwinter, and see whether I understood correctly.

Reply:
Yes. It has no difference if we wrote 2016 or 2018 here. Because this hook will get the date from the os based on its first created time. 
https://github.com/PaddlePaddle/Paddle/pull/7906/files#diff-dcba310b4875a3619fb0ffd77ef4fbc1R39

Reply:
@sidgoyal78 Your understanding is correct. There is no difference between 2016 and 2018 in the copyright template.
This copyright hook will get the creation date of a file. https://github.com/PaddlePaddle/Paddle/pull/7906/files#diff-dcba310b4875a3619fb0ffd77ef4fbc1R39

Reply:
When would this script file be applied? Is it applied to new source code files only, or is it applied to all files changed in a commit, or something else?

Reply:
@wangkuiyi  It is applied to new source code files only.